src/backend/expungeservice/record_summarizer.py

```diff
@@ -86,3 +91,18 @@ def get_label(charge: Charge):
             ]
             eligible_charges_by_date[label] = charges_tuples
         return eligible_charges_by_date
+
+    @staticmethod
+    def _build_is_all_cases_feeless(charges):
+        return bool(
+            charges
```

Comment:
Should this be charges or eligible charges?

Reply:
The method checks if there are eligible charges without fees, which is the circumstance that the "no convictions" message is shown in the fees section.